scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -39,6 +39,9 @@
 parser.add_argument("--selectNonPromptFromLightMesonDecay", action="store_true", help="Test: define a non-prompt muon enriched control region with muons from light meson decays")
 parser.add_argument("--useGlobalOrTrackerVeto", action="store_true", help="Use global-or-tracker veto definition and scale factors instead of global only")
 parser.add_argument("--useRefinedVeto", action="store_true", help="Temporary option, it uses a different computation of the veto SF (only implemented for global muons)")
+parser.add_argument("--noVetoSF", action="store_true", help="Don't use SF for the veto, for tests")
+parser.add_argument("--scaleDYvetoFraction", type=float, default=-1.0, help="Scale fraction of DY background that should receive veto SF by this amount. Negative values do nothing")
+parser.add_argument("--scaleDYfull", type=float, default=-1.0, help="Scale inclusive DY background (as if scaling the cross section). Negative values do nothing")
```

Comment:
Scaling the inclusive DY background can also be done after the histograms are produced, e.g. in loadHistsForDatagroups with small modifications (by changing the cross section for example)
If there is no other reason I think it should be done there

Reply:
Ok, I'll remove the option (I don't think it's necessary to reimplement this feature in setupCombine.py right now, we can think of it in a second moment)